### PR TITLE
[GSW-416] feat: Integrate Pools API

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: packages/web/.env.example [hunk 1]
```diff
@@ -1 +1,5 @@
-NEXT_PUBLIC_API_URL="https://mock-api.gnoswap.io/v3/testnet"
\ No newline at end of file
+NEXT_PUBLIC_API_URL="https://dev-api.gnoswap.io/v3/testnet"
+NEXT_PUBLIC_PACKAGE_POOL_PATH="gno.land/r/pool"
+NEXT_PUBLIC_PACKAGE_POOL_ADDRESS="g1ee305k8yk0pjz443xpwtqdyep522f9g5r7d63w"
```
Comment (harryoh) [inline packages/web/.env.example:3]: It has `POOL_PATH`, but does it need to know the `address`? Is there any way to find out?

Comment (harryoh) [reply packages/web/.env.example:3]: But if the task makes more complicated, it may be okay to just use two variables.

Comment (jinoosss) [reply packages/web/.env.example:3]: We can implement an algorithm that generates an address from a path.
However, we must utilize a crypto library such as bech32.

I'm not sure which one is better, what do you think?

Comment (harryoh) [reply packages/web/.env.example:3]: First, let's use two variables.

**Review by harryoh — APPROVED** [review]
👍

**Pull request merged**